pkg/api/types.go

```diff
@@ -144,18 +157,20 @@ type InstanceSpec struct {
 	// CF-specific; move to annotations
 	CFCredentials   string
 	CFDashboardURL  string
-	CFInternalID    string // came from ville. remove? nobody likes that guy
```

Comment:
this was a mega joke from the Boulder face-to-face meeting. for the record, I believe that everybody likes @vaikas-google!